forge-gui/res/cardsfolder/upcoming/become_the_pilot.txt

```diff
@@ -0,0 +1,9 @@
+Name:Become the Pilot
+ManaCost:3 U U
+Types:Enchantment Aura
+K:Enchant noncommander creature
+A:SP$ Attach | ValidTgts$ Creature.IsNotCommander | TgtPrompt$ Select target noncommander creature | AILogic$ GainControl
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | GainControl$ You | Description$ You control enchanted creature.
```

Comment:
@Hanmac
the GainControl$ could be merged with the other or does it matter?

Reply:
From the oracle it's two abilities